test/e2e/framework/util.go

```diff
@@ -4847,7 +4847,7 @@ func getMaster(c clientset.Interface) Address {
 		Failf("Failed to get kubernetes endpoints: %v", err)
 	}
 	if len(eps.Subsets) != 1 || len(eps.Subsets[0].Addresses) != 1 {
-		Failf("There are more than 1 endpoints for kubernetes service: %+v", eps)
+		Failf("There is more than 1 endpoint for kubernetes service: %+v", eps)
```

Comment:
both the new and old forms are grammatically correct.
please leave the original.